src/stirling/source_connectors/socket_tracer/protocols/mongodb/types.h

```diff
@@ -116,6 +121,9 @@ constexpr int32_t kMaxBSONObjSize = 16000000;
  * https://github.com/mongodb/specifications/blob/e09b41df206f9efaa36ba4c332c47d04ddb7d6d1/source/message/OP_MSG.rst#command-arguments-as-payload
  *
  * There can be 0 or more documents in a section of kind 1 without a separator between them.
+ *
+ * Information about MongoDB handshaking messages can be found here:
+ * https://github.com/mongodb/specifications/blob/master/source/mongodb-handshake/handshake.rst
```

Comment:
Can we make this a permalink so that any future changes to the file will link to the current copy?